l/llvmlite/build_info.json

```diff
@@ -1,15 +1,23 @@
 {
-  "maintainer": "stutiibm",
-  "package_name": "llvmlite",
-  "github_url": "https://github.com/numba/llvmlite",
-  "version": "v0.41.1",
-  "default_branch": "main",
-  "package_dir": "l/llvmlite",
-  "build_script": "llvmlite_ubi_8_7.sh",
-  "docker_build": false,
-  "validate_build_script": true,
-  "use_non_root_user": false,
-  "v0.41.1, v0.41.0": {
-    "build_script": "llvmlite_ubi_8_7.sh"
-  }
-}
\ No newline at end of file
+   "maintainer": "npanpaliya",
+   "package_name": "llvmlite",
+   "github_url": "https://github.com/numba/llvmlite",
+   "version": "*",
+   "default_branch": "main",
+   "build_script": "llvmlite_v0.41.1_ubi_9_3.sh",
+   "package_dir": "l/llvmlite",
+   "docker_build": true,
+   "validate_build_script": true,
+   "use_non_root_user": false,
+   "*": {
```

Comment:
If no docker build support you can remove this whole block  
   "*": {
     "dir": "linux",
      "base_docker_image": "base",
      "base_docker_variant": "Not defined"
   },